src/main/java/com/minecolonies/coremod/colony/buildings/workerbuildings/BuildingHome.java

```diff
@@ -45,6 +46,27 @@
     @NotNull
     private final List<BlockPos> bedList = new ArrayList<>();
 
+    /**
+     * Is a female citizen living here?
+     */
+    private boolean femalePresent = false;
+
+    /**
+     * Is a male citizen living here?
+     */
+    private boolean malePresent = false;
+
+    /**
+     * Interval at which the childen are created, in ticks.
+     * Every 20 min it tries to spawn a child, 20min*60s*20ticks
+     */
+    private int childCreationInterval = 2 * 60 * 20;
```

Comment:
Isn't that a constant?

Reply:
it is modifyable for future changes to the interval, e.g. for a reasearch from the university. going to change this to the spawn chance for the random